packages/bff/src/routers/catalogRouter.ts

```diff
@@ -134,7 +137,21 @@ const catalogRouter = (
     )
     .post(
       "/eservices/:eServiceId/descriptors/:descriptorId/documents",
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        const ctx = fromBffAppContext(req.ctx, req.headers);
+        try {
+          const resp = await catalogService.createEServiceDocument(
+            req.params.eServiceId,
+            req.params.descriptorId,
+            req.body,
+            ctx
+          );
+          return res.status(200).json(resp).send();
+        } catch (error) {
+          const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
```

Comment:
This endpoint should have a dedicated mapper, with the errors that you added, and an operational log